.github/workflows/ci-aarchxx-cross.yml

```diff
@@ -98,30 +98,6 @@ jobs:
         CI_TRIGGER: ${{ github.event_name }}
```

Comment:
> we skip filtering by test suite status mostly for simplicity of logic that must be duplicated in each workflow file.

What is this referring to: what filtering?  Is this referring to the fact that we had a separate dedicated email for each job within a file, while now there's one that lists the status of all the jobs?  Please clarify.

Reply:
> we skip filtering by test suite status mostly for simplicity of logic that must be duplicated in each workflow file.

See part before semi-colon: "Status of all the test suite jobs are sent".

Did s/skip filtering by/skip filtering of status message by/